client/src/main/java/com/vesoft/nebula/client/graph/net/NebulaPool.java

```diff
@@ -177,7 +179,7 @@ public int getWaitersNum() {
      * Update the services' status when the connection is broken,
      * it is called by Session and NebulaPool
      */
-    protected void updateServerStatus() {
+    protected void updateServerStatus() throws ClientServerIncompatibleException {
```

Comment:
other interfaces should not throw ClientServerIncompatibleException except getSession. Because once the connection has been connected correct, other operations would not throw ClientServerIncompatibleException ever.